### PR TITLE
feature/QOS-1-add-new-keybinding: Add CTRL + E shortcut

### PR DESCRIPTION
**Pull request by georgiai1**
adding the same functionality to the other quickOpen shortcut CTRL + E

### CODE REVIEW

**Comment by finico** [comment]
Hi @gkkirilov ! I appreciate the PR!
Why you need this shortcut with same functionality?
In the settings I can see that this shortcut is reserved for the search by default.

screen shot 2017-12-01 at 12 47 44

**Comment by georgiai1** [comment]
Thanks for the quick reply @finico!
Yeah I am on windows so the CTRL + E is the same as CTRL + P both open quickOpen,
Should I change the PR to affect windows shortcuts only?

**Comment by finico** [comment]
@gkkirilov Yes. It would be nice. This is necessary in order not to worsen the usual UX for mac users.

**Comment by georgiai1** [comment]
Thanks for feedback @finico , fixed!

File: package.json [hunk 1]
```diff
@@ -43,6 +43,11 @@
 				"key": "ctrl+p",
 				"mac": "cmd+p",
 				"when": "editorTextFocus"
+			},
+			{
+				"command": "quickOpenSelection.quickOpenSelection",
+				"key": "ctrl+e",
```
Comment (finico) [inline package.json]: `key` applies to all platforms.
It should be `"win": "ctrl+e"`

Comment (georgiai1) [reply package.json]: Fixed

**Comment by finico** [comment]
@gkkirilov released!

**Comment by georgiai1** [comment]
Hey @finico I didnt have time to try it but today when I build it,it requires the "key": property in the keybinding and the command didnt work. Can you try building it like that on the mac and see if it overwrites it? Because I don't have mac near me
{
	"command": "quickOpenSelection.quickOpenSelection",
	"key": "ctrl+e",
	"win": "ctrl+e",
	"when": "editorTextFocus"
}

**Comment by finico** [comment]
Hi @gkkirilov Yes, it overrides.
I tried this:
```json
{
  "command": "quickOpenSelection.quickOpenSelection",
  "key": "ctrl+e",
  "mac": "cmd+p",
  "when": "editorTextFocus"
}
```
It works fine.
I checked default keys on linux and it the same as on windows.
And all is well in this case.
You can make a PR or I'll fix it soon.

**Pull request merged**